apps/dashboard/src/app/AppHeader.tsx

```diff
@@ -30,6 +35,10 @@ export class AppHeader extends React.Component<IAppHeaderProps> {
   private readonly onDatasetSelect = this.onSelect.bind(this, "dataset");
   private readonly onThemeSelect = this.onSelect.bind(this, "theme");
   private readonly onLanguageSelect = this.onSelect.bind(this, "language");
+  private readonly onFeatureFlightSelect = this.onFlightSelect.bind(
```

Comment:
this was lagacy code, bind is not good practice here, we are moving to arrow function, do you mind update all these to be arrow functions?

Reply:
I remember you telling me that :-) I was surprised to see it here, so I didn't want to do something different, but let me change this!